src/structures/Guild.js

```diff
@@ -144,22 +201,38 @@ class Guild extends Base {
     this.discoverySplash = data.discovery_splash;
 
     /**
-     * The region the guild is located in
-     * @type {string}
+     * The embed channel ID, if enabled
+     * @type {?string}
+     * @deprecated
      */
-    this.region = data.region;
+    this.embedChannelID = data.embed_channel_id;
 
     /**
-     * The full amount of members in this guild
-     * @type {number}
+     * Whether embedded images are enabled on this guild
+     * @type {boolean}
+     * @deprecated
      */
-    this.memberCount = data.member_count || this.memberCount;
+    this.embedEnabled = data.embed_enabled;
+
+    if (!this.emojis) {
```

Comment:
Why is this not always present, like channels or roles or members?

Reply:
on the inital call its not present

Reply:
on the initial call its not present